### PR TITLE
update bug report template

### PR DESCRIPTION
**Pull request by pascalkuthe**
Added some hints to the bug report template to (hopefully) reduce the number of inactionable bug reports.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ body:
     id: helix-log
     attributes:
       label: Helix log
-      description: See `hx -h` for log file path
+      description: See `hx -h` for log file path, if you can reproduce the issue run `RUST_BACKTRACE=1 helix -v` to generate a more detailed log file.
```
Comment (the-mikedavis) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:35]:
```suggestion
      description: See `hx -h` for log file path. If you can reproduce the issue run `RUST_BACKTRACE=1 hx -v` to generate a more detailed log file.
```

Comment (dead10ck) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:35]: Also, I'd err on the side of more verbosity with `-vvvv`

**Review by dead10ck — APPROVED** [review]
Lgtm!

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -32,7 +32,7 @@ body:
     id: helix-log
     attributes:
       label: Helix log
-      description: See `hx -h` for log file path
+      description: See `hx -h` for log file path. If you can reproduce the issue run `RUST_BACKTRACE=1 hx -vvvv` to generate a more detailed log file.
```
Comment (archseer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:35]: vvvv is a bit extreme since you'll get logs for tokio internals. Maybe v or vv?

Comment (pascalkuthe) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: I changed it to `vv`. I originally used `v` but @dead10ck suggested the more verbose version. I agree that including tracing output is probably too much so `vv` seems more appropriate.

**Review by archseer — APPROVED** [review]

**Pull request merged**